src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Serialization/SqliteJournalSerializationSpec.cs

```diff
@@ -40,5 +47,15 @@ class = ""Akka.Persistence.Sqlite.Journal.SqliteJournal, Akka.Persistence.Sqlite
                     }
                 }");
         }
+
+        [Fact(Skip = "Sql plugin does not support SerializerWithStringManifest")]
+        public override void Journal_should_serialize_Persistent_with_string_manifest()
```

Comment:
Yes, it should. `SerializerWithStringManifest` was designed especially for Persistence. But it is not critical and easy to implement.